wrangler-core/src/test/java/io/cdap/wrangler/TestingPipelineContext.java

```diff
@@ -128,4 +129,24 @@ public TransientStore getTransientStore() {
   public <T> Lookup<T> provide(String s, Map<String, String> map) {
     return null;
   }
+
+  /**
+   * Gets the child metrics by adding specified tags to the current metrics context.
+   * @param tags The tags be added
+   * @return A Metrics instance
+   */
+  @Override
+  public Metrics child(Map<String, String> tags) {
```

Comment:
I thought compilation was failing for here - https://github.com/data-integrations/wrangler/pull/458/files#diff-441e5f59633fc5af8083c71f20498613f58bdccc3ab7583515e4bf9c71a3bb20R65 where returned StageMetrics does not implement additional methods?

Reply:
I think I misread something in the original class, thanks for the heads up.